src/__tests__/CompareResults/SubtestsResultsView.test.tsx

```diff
@@ -0,0 +1,228 @@
+      route: '/subtestsCompareWithBase',
+      search:
+        '?baseRev=d775409d7c6abb76362a3430e9880ec032ad4679&baseRepo=mozilla-central&newRev=22f4cf67e8ad76b5ab2a00b97837d1d920b8c2b7&newRepo=mozilla-central&framework=1&baseParentSignature=4769486&newParentSignature=4769486',
+      loader,
+    });
+
+    const subTestsMain = await screen.findByTestId('subtests-main');
+
+    expect(subTestsMain).toBeInTheDocument();
+  });
+
+  it('should confirm that the subtests heading is present', async () => {
+    const { subtestsResult } = getTestData();
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://treeherder.mozilla.org/api/perfcompare/results/',
+      subtestsResult,
+    );
+    renderWithRouter(<SubtestsResultsMain view={'subtests-results'} />, {
+      route: '/subtestsCompareWithBase',
+      search:
+        '?baseRev=d775409d7c6abb76362a3430e9880ec032ad4679&baseRepo=mozilla-central&newRev=22f4cf67e8ad76b5ab2a00b97837d1d920b8c2b7&newRepo=mozilla-central&framework=1&baseParentSignature=4769486&newParentSignature=4769486',
+      loader,
+    });
+
+    const subTestsHeading = await screen.findByText('Subtests');
+
+    expect(subTestsHeading).toBeInTheDocument();
+  });
+
+  it('should display breadcrumbs for easy navigation', async () => {
+    const { subtestsResult } = getTestData();
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://treeherder.mozilla.org/api/perfcompare/results/',
+      subtestsResult,
+    );
+    renderWithRouter(<SubtestsResultsMain view={'subtests-results'} />, {
+      route: '/subtestsCompareWithBase',
+      search:
+        '?baseRev=d775409d7c6abb76362a3430e9880ec032ad4679&baseRepo=mozilla-central&newRev=22f4cf67e8ad76b5ab2a00b97837d1d920b8c2b7&newRepo=mozilla-central&framework=1&baseParentSignature=4769486&newParentSignature=4769486',
+      loader,
+    });
+
+    const homeBreadcrumb = await screen.findByText('Home');
+    const resultsBreadcrumb = await screen.findByText('All results');
+
+    expect(homeBreadcrumb).toBeInTheDocument();
+    expect(resultsBreadcrumb).toBeInTheDocument();
+  });
+
+  it('should confirm that the subtests results table is displayed', async () => {
+    const { subtestsResult } = getTestData();
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://treeherder.mozilla.org/api/perfcompare/results/',
+      subtestsResult,
+    );
+    renderWithRouter(<SubtestsResultsMain view={'subtests-results'} />, {
+      route: '/subtestsCompareWithBase',
+      search:
+        '?baseRev=d775409d7c6abb76362a3430e9880ec032ad4679&baseRepo=mozilla-central&newRev=22f4cf67e8ad76b5ab2a00b97837d1d920b8c2b7&newRepo=mozilla-central&framework=1&baseParentSignature=4769486&newParentSignature=4769486',
+      loader,
+    });
+
+    const table = await screen.findByTestId('subtests-results-table');
+
+    expect(table).toBeInTheDocument();
+  });
+
+  it('should request authorization code when "Retrigger" button is clicked', async () => {
```

Comment:
Isn't this already tested in Retrigger.test.jsx?

Reply:
No, it's not tested in `Retrigger.test.jsx`. However, it is covered in `ResultsView.test.tsx`, where the retrigger button for the results view is tested.
I have tested the retrigger button just the same but for subtests view in `SubtestsResultsView.test.tsx`